### ISSUE TITLE
Accessibility: Volume slider is not visible under Windows 10’s “High-contrast” mode

### ISSUE THREAD

**Issue by KNTRO** [mastodon/mastodon#38989]
### Steps to reproduce the problem

1. Enable the “High-contrast” mode on Windows 10. To do so: Windows key + I, go to the “Accessibility” section > at the left menu click on “High-contrast” > then enable it by using the toggle and select the “High-contrast 1” theme.
2. Run your web browser and check if it detects Windows’ “High-contrast” mode automatically. If not, go to your web browser settings menu and search for “accessibility” or “high contrast”. If you find an option to use the color schema by the system, use it.
3. Now, you need to go to a Mastodon post with media attached (audio or video), and you will find out that the volume slider is invisible.

### Expected behaviour

The volume slider should be visible

### Actual behaviour

The volume slider is not visible

### Mastodon instance

mas.to

### Mastodon version

v4.5.9

### Browser name and version

Firefox 140.0 ESR

### Operating system

Windows 10

### Technical details

Screenshot attached.

<img width="1502" height="706" alt="Image" src="https://github.com/user-attachments/assets/8fc94666-daf4-48b9-91c7-4232d2ceb55f" />

**Comment by KNTRO** [comment]
Toggles are not visible either.

<img width="604" height="621" alt="Image" src="https://github.com/user-attachments/assets/8fc1eb44-08ec-495a-a35e-bf395b2730d5" />

**Comment by diondiondion** [comment]
Thanks for reporting these! We hope to be able to look into these for the upcoming 4.6 release

**Comment by KNTRO** [comment]
@diondiondion Thank you so much for your invaluable help! I really appreciate your interest solving this issue! 🙏
